bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -72,7 +73,7 @@ public void run() {
             int size = ACTIVE_BUFFERED_MUTATORS.size();
             new Logger(AbstractBigtableConnection.class).warn(
               "Shutdown is commencing and you have open %d buffered mutators."
-                  + "You need to close() or flush() them so that is not lost", size);
+                  + "You need to close() or awaitCompletion() them so that is not lost", size);
```

Comment:
This message should still be flush(), since it refers to the buffered mutator and not the RpcThrottler.